### ISSUE TITLE
Is my label pre-training normal?

### ISSUE THREAD

**Issue by yyf17** [facebookresearch/sound-spaces#55]
@ChanganVR 
Thank you, I appreciate your wonderful work.

I tried to rerun your **_savi_** experiments following the usage you wrote. I have 1 GPU.

The accuracy of my training curve was high at first, but it has been declining since then.

![image](https://user-images.githubusercontent.com/34536352/133744197-2dd43911-105f-4bd2-9d68-3f50c439119e.png)

![image](https://user-images.githubusercontent.com/34536352/133744048-e0928eb2-8402-4639-a9a9-a5a40862ae57.png)

![image](https://user-images.githubusercontent.com/34536352/133744434-ee59f553-6fee-4a64-bdbb-cd1032c7828c.png)

![image](https://user-images.githubusercontent.com/34536352/133744514-4707d806-22e3-4717-a8de-b5245eceb306.png)
The training parameters have not been modified.

![image](https://user-images.githubusercontent.com/34536352/133745207-3891e5f1-b867-49a4-92d9-1a8dab7bcd75.png)

Is my label pre-training normal?

**Comment by ChanganVR** [comment]
This is weird. Did you use the pretraine weights for training? Why would the training classifier accuracy start at such a high value?

When I trained label predictor, the performance increased over time and converged eventually.

**Comment by yyf17** [comment]
Thank you, I will try again.

**Comment by sivannavis** [comment]
Hi @yyf17 @ChanganVR  thanks for the discussion, hope you are doing good! I'm new to this repo and had some trouble training this label predictor with the given parameter settings and hope to get some suggestions. I'm in the same situation here with accuracy, @yyf17 could you give some tips about what is the problem?
<img width="1424" alt="image" src="https://user-images.githubusercontent.com/50509896/184265431-7de3ec15-ecdb-476e-a30d-84a5a3d012fe.png">

Also about runtime, I'm wondering why the first epoch is so slow with 20% cpu util and 0% gpu util. In other epochs, cpu util stays around 100% but gpu util keeps jumping between 0 to 100%. I'm training via Slurm with 1 node, 1024 batch size and 8 workers. I guess it has something to do with the cpu IO bottleneck in dataloader? Do you have any workaround advice?
